Sources/TSPL/TSPL.docc/LanguageGuide/Initialization.md

```diff
@@ -2,6 +2,8 @@
 
 # Initialization
 
+Set the initial values for a type's stored properties.
```

Comment:
```suggestion
Set the initial values for a type's stored properties and perform one-time setup.
```

Seems like initializers are also commonly used to perform setup or processing operations that only need to happen once, once you have the initial data to work on.